src/server/common/test-helpers/csrf.js

```diff
@@ -0,0 +1,76 @@
+import { statusCodes } from '~/src/server/common/constants/status-codes.js'
+/* global expect, test */
+
+/**
+ * @param {OutgoingHttpHeaders} headers
+ * @param {string} name
+ * @returns {string}
+ */
+const findCookie = (headers, name) => {
+  const setCookie = headers['set-cookie']
+
+  /** @type {string[] | undefined} */
+  const cookies = typeof setCookie === 'string' ? [setCookie] : setCookie
```

Comment:
given you are wanting to put it into an array
would `Array.isArray` not be more appropriate?

Reply:
If it's not an array, it's either undefined or a string, so I wrapped it in an array if it .. isn't an array, then we'd end up with `{string[] | undefined[]}` or something similar - which just felt odd.

But it's ugly as is, so I'm going pushing an alternative that's more direct.